.rubocop_local.yml

```diff
@@ -0,0 +1,19 @@
+#
+# Overrides
+#
+GlobalVars:
+  AllowedVariables:
```

Comment:
Should these loggers be allowed?  I think we should encourage manageiq-smartstate to add a logger and log to that instead of continuing to use `$log` and `$vim_log`.  Thoughts?